cli/stream.py

```diff
@@ -375,6 +400,9 @@ def stream(
         batch_web3_provider=ThreadLocalProxy(lambda: get_provider_from_uri(provider_uri, batch=False)),
         job_scheduler=job_scheduler,
         sync_recorder=create_recorder(sync_recorder, config),
+        limit_reader=create_limit_reader(
```

Comment:
what if user put other value in source_path?

Reply:
In unexpected cases, limit_reader will request the latest block from rpc.
if source_path specify a csv_source, parameter checking would ensure that the rpc_limit_reader does not interfere with the program